build-with-tag.sh

```diff
@@ -33,8 +33,10 @@ git fetch --all
 git merge
 
 scalas=$(awk '/scala_version=/' < ./Dockerfile | cut -c 19-)
-scala=$(echo $scalas | awk '{split($0,a," "); print a[1]}')
-kafka=$(awk '/kafka_version=/' < ./Dockerfile | cut -c 19-)
+# I guess the first occurence is enough as it doesn't make much sense I think to use
+# different scala and kafka versions between the images.
```

Comment:
Since Apache Kafka 3.0.0, Scala 2.12 has been deprecated so we don't need to support building docker images for both Scala 2.13 and 2.12 any more

Reply:
I think you misunderstood my comment, it's more about how the Dockerfile has 2 images as bases in it and we grab some stuff from the first and put it in the second one, which is fine, I'm just guessing that it doesn't make sense to have the first image as for example Kafka 3.0.0 and the second one as 3.1.0.